app/controllers/static_pages_controller.rb

```diff
@@ -1,4 +1,4 @@
-class PagesController < ApplicationController
+class StaticPagesController < ApplicationController
```

Comment:
Does `ApplicationController` get extended by high_voltage? Or should this be subclassing a controller provided by high_voltage?

Reply:
this controller is separate from HighVoltage, by default HV uses `PagesController`, which is why I renamed this existing controller